parm/FV3LAM_wflow.xml

```diff
@@ -922,22 +930,26 @@ the <task> tag to be identical to the ones above for other output times.
     {%- endif %}
 
     <dependency>
-      {%- if run_task_get_obs_ccpa %}
       <and>
-        <taskdep task="&GET_OBS_CCPA_TN;"/>
+{#- Redundant dependency to simplify jinja code. #}
+        <streq><left>TRUE</left><right>TRUE</right></streq>
+      {%- if run_task_get_obs_ccpa %}
+        <and>
+          <taskdep task="&GET_OBS_CCPA_TN;"/>
         {%- if write_dopost %}
-        <taskdep task="&RUN_FCST_TN;{{ uscore_ensmem_name }}"/>
-        {%- else %}
-        <metataskdep metatask="&RUN_POST_TN;{{ uscore_ensmem_name }}"/>
+          <taskdep task="&RUN_FCST_TN;{{ uscore_ensmem_name }}"/>
+        {%- elif run_task_run_post %}
+          <metataskdep metatask="&RUN_POST_TN;{{ uscore_ensmem_name }}"/>
         {%- endif %}
-      </and>
+        </and>
       {%- else %}
         {%- if write_dopost %}
-      <taskdep task="&RUN_FCST_TN;{{ uscore_ensmem_name }}"/>
-        {%- else %}
-      <metataskdep metatask="&RUN_POST_TN;{{ uscore_ensmem_name }}"/>
+        <taskdep task="&RUN_FCST_TN;{{ uscore_ensmem_name }}"/>
+        {%- elif run_task_run_post %}
+        <metataskdep metatask="&RUN_POST_TN;{{ uscore_ensmem_name }}"/>
         {%- endif %}
       {%- endif %}
+      </and>
```

Comment:
It seems like this logic can be simplified to the following:

```
      <and>
{#- Redundant dependency to simplify jinja code. #}
        <streq><left>TRUE</left><right>TRUE</right></streq>
      {%- if run_task_get_obs_ccpa %}
        <taskdep task="&GET_OBS_CCPA_TN;"/>
      {%- endif %}
      {%- if write_dopost %}
        <taskdep task="&RUN_FCST_TN;{{ uscore_ensmem_name }}"/>
      {%- elif run_task_run_post %}
        <metataskdep metatask="&RUN_POST_TN;{{ uscore_ensmem_name }}"/>
      {%- endif %}
      </and>
```
The nested "and"s are superfluous, and `if run_task_get_obs_ccpa` does not require an "else" statement.

The same logic should apply to the subsequent changes for `run_task_get_obs_mrms` and `run_task_get_obs_ndas` as well.

Reply:
@mkavulich Yes I noticed that too but had decided to make only the minimal change necessary (in terms of when you do a diff) since all this will get rewritten anyway pretty soon.  I'll put in your suggestion and rerun.

Reply:
@mkavulich I put in the new simpler logic and reran the vx tests (with intel compiler only) and all 3 succeeded.  The other (fundamental) tests aren't affected by this, so I didn't rerun them.  I'll let the Jenkins testing handle those.